modules/dnn/src/layers/scale_layer.cpp

```diff
@@ -26,17 +26,37 @@ class ScaleLayerImpl : public ScaleLayer
     {
         setParamsFrom(params);
         hasBias = params.get<bool>("bias_term", false);
+
+        if (blobs.empty())
```

Comment:
We can add an extra layers in `.prototxt` that were not used during training. So they have no weights. In this model it's a Scale layer.